pkg/proxy/iptables/proxier.go

```diff
@@ -1518,12 +1518,22 @@ func (proxier *Proxier) syncProxyRules() {
 	// Drop the packets in INVALID state, which would potentially cause
 	// unexpected connection reset.
 	// https://github.com/kubernetes/kubernetes/issues/74839
-	writeLine(proxier.filterRules,
-		"-A", string(kubeForwardChain),
-		"-m", "conntrack",
-		"--ctstate", "INVALID",
-		"-j", "DROP",
-	)
+	if proxier.localDetector.IsImplemented() {
```

Comment:
What I don't understand is why This wouldn't experience https://github.com/kubernetes/kubernetes/issues/74839 for non-local connections?